Spartan/spartan.munki.recipe

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+	<string>Downloads the latest version of Spartan from www.wavefun.com and imports it into Munki</string>
+	<key>Identifier</key>
+	<string>com.github.its-unibas.munki.spartan</string>
+	<key>Input</key>
+	<dict>
+		<key>MUNKI_REPO_SUBDIR</key>
+		<string>apps/spartan</string>
+		<key>pkginfo</key>
+		<dict>
+			<key>catalogs</key>
+			<array>
+				<string>testing</string>
+			</array>
+			<key>category</key>
+			<string>Productivity</string>
+			<key>description</key>
+			<string>Desktop application for chemistry research and academia.</string>
+			<key>developer</key>
+            <string>Wavefunction Inc.</string>
+            <key>minimum_os_version</key>
+            <string>10.12.0</string>
+            <key>unattended_install</key>
+            <true />
+		</dict>
+	</dict>
+	<key>MiniumumVersion</key>
+	<string>1.0</string>
+	<key>ParentRecipe</key>
+	<string>com.github.its-unibas.download.spartan</string>
+	<key>Process</key>
+	<array>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>input_plist_path</key>
+				<string>%pathname%/%app_name%/Contents/Info.plist</string>
+				<key>plist_version_key</key>
+				<string>CFBundleVersion</string>
+			</dict>
+			<key>Processor</key>
+			<string>Versioner</string>
+		</dict>
+		<dict>
+			<key>Arguments</key>
+			<dict>
+				<key>plist_keys</key>
+				<dict>
+					<key>CFBundleName</key>
+					<string>name</string>
+					<key>CFBundleShortVersionString</key>
+					<string>display_name</string>
```

Comment:
While perfectly valid, this is (for me) an unusual way to generate Name/DisplayName. Could you do the following:
* Name should not be derived automatically. All munki actions trigger on the name variable and I don't want this to be at the developers mercy
* I'm okay with deriving the display name this way. Could you add a short note about why you chose this in the recipe description?